core/electron-manager/src/common/ElectronRpcManager.ts

```diff
@@ -10,7 +10,7 @@ import { ElectronRpcProtocol } from "./ElectronRpcProtocol";
 import { IpcSocket, IpcSocketBackend, IpcSocketFrontend, RpcConfiguration, RpcInterfaceDefinition, RpcManager } from "@bentley/imodeljs-common";
 
 /** RPC interface configuration for an Electron-based application.
- * @internal
+ * @beta
```

Comment:
@kabentley I had to switch this API to `beta` as it's a part of the constructor of `ElectronRpcProtocol` which is in `beta`.

Reply:
probably it should all be switched to internal

Reply:
Even `ElectronRpcRequest` or should i just make the `ElectronRpcRequest.protocol` internal?

Reply:
yes, `ElectronRpcRequest` should be internal

Reply:
Sounds good, just pushed that change.  Hoping it fixes the doc linking issue with ElectronHost.

Reply:
That didn't fix the docs issues but I did talk to @williamkbentley and he told me what the issue is.  It will be a bit larger of a change so I'll do it in a separate PR.